### ISSUE TITLE
no-invalid-triple-slash-reference does not like no-default-lib="true"

### ISSUE THREAD

**Issue by lucacasonato** [denoland/deno_lint#938]
### Lint Name

no-invalid-triple-slash-reference

### Code Snippet

JS file:

```js
/// <reference no-default-lib="true"/>
```

### Expected Result

No lint diagnostic

### Actual Result

```
(no-invalid-triple-slash-reference) This triple-slash reference directive is not allowed in JavaScript
/// <reference no-default-lib="true"/>
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

### Version

1.15.1
